.github/workflows/e2e.yaml

```diff
@@ -35,3 +35,12 @@ jobs:
       env:
         KUBE_VERSION: ${{ matrix.kube-version }}
       run: make prepare-e2e e2e KUBE_VERSION=$KUBE_VERSION
+
+    - uses: jpkrohling/setup-operator-sdk@v1.1.0
+        with:
+          operator-sdk-version: v1.13.1
+
+    - name: "run scorecard test"
```

Comment:
Perhaps split this into its own workflow so that it can be restarted individually? You can externalize the Kubernetes matrix and use it as input here. If you can't find the docs for this, let me know.

Reply:
send me the docs

Reply:
https://docs.github.com/en/actions/learn-github-actions/reusing-workflows

Reply:
I could not make it work in a nice way. 

![Screenshot of Fix · open-telemetry_opentelemetry-operator@3a2c768](https://user-images.githubusercontent.com/5618424/141481588-62f1eab5-5dc0-42c7-ba95-535d67330974.jpg)
![Screenshot of Fix · open-telemetry_opentelemetry-operator@7b3efcc](https://user-images.githubusercontent.com/5618424/141481596-94782101-8024-4185-8ad8-787858d6e9a7.jpg)